SonarQube.TeamBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -97,8 +97,8 @@ public bool Execute(ILogger logger, string projectKey, string projectName, strin
 
             // Create the directories
             logger.LogMessage(Resources.DIAG_CreatingFolders);
-            EnsureEmptyDirectory(logger, config.SonarConfigDir);
-            EnsureEmptyDirectory(logger, config.SonarOutputDir);
+            Utilities.EnsureEmptyDirectory(config.SonarConfigDir, logger);
+            Utilities.EnsureEmptyDirectory(config.SonarOutputDir, logger);
 
```

Comment:
I changed the parameter ordering - makes more sense to have the name of the directory being created first.